test/opentype.spec.mjs

```diff
@@ -84,23 +57,6 @@ describe('opentype.js', function() {
         assert.equal(aGlyph.path.commands.length, 14);
     });
 
-    it('[deprecated] .load() handles a parseBuffer error', function(done) {
```

Comment:
I thought we agreed that this would still function in 2.0.0 with a warning, and be removed in a subsequent release.

Reply:
the 2.0.0 is a major release
major release does break things
so if we are brewing the 2.0.0 it make more sense to put the warning now to ease this major transition